userguide/drawing_classifier/export-coreml.md

```diff
@@ -29,7 +29,27 @@ the drawing you want to classify, you can use the Vision framework to consume
 the exported Core ML model. Note that the image you provide to the Vision API
 must be a Grayscale Image. 
 
+```swift
+let model = try VNCoreMLModel(for: MySquareTriangleClassifier().model)
+
+let request = VNCoreMLRequest(model: model, completionHandler: { [weak self] request, error in
+    self?.processClassifications(for: request, error: error)
+})
+
+let grayscaleImage = UIImage(named: bitmapFilename)
+let handler = VNImageRequestHandler(cgImage: grayscaleImage.cgImage!, options: [:])
+
+try? handler.perform([request])}
+
+func processClassifications(for request: VNRequest) {
```

Comment:
I think you're missing an error parameter. I think this line should be:
`func processClassifications(for request: VNRequest, error: Error?) {`

Let's put this func in its own code block to make it clear that it's a separate function.